### PR TITLE
Add Shepherds Pie

### PR DESCRIPTION
**Pull request by HazmatDrone**

### CODE REVIEW

File: Spaghetti-Bolognese.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Spaghetti Bolognese
```
Comment (pacien) [inline Spaghetti-Bolognese.md:1]: That file/recipe shouldn't be included in this pull request.

File: shepherds-pie [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Shepherd's Pie
```
Comment (pacien) [inline shepherds-pie:1]: This file's name lacks the proper extension.

**Review by pacien — CHANGES_REQUESTED** [review]

File: Spaghetti-Bolognese.md (hunk 1 above)
Comment (Krutonium) [reply Spaghetti-Bolognese.md:1]: Agreed, Please Amend

File: shepherds-pie (hunk 2 above)
Comment (Krutonium) [reply shepherds-pie:1]: Also this.
